packages/dates/src/config.ts

```diff
@@ -6,6 +7,7 @@ export type ISOWeek = string; // isoweek, e.g. '2024-W01'
 
 export type SeasonConfig = {
   title: string;
+  id: string;
```

Comment:
MUST FIX: we already use the 'start' as the pointer to seasons. this jsut introduces a brand new way to reference seasons and i dont think that is helpful